dts/bindings/input/seeed,chsc6x.yaml

```diff
@@ -0,0 +1,14 @@
+description: CHSC6X touchscreen sensor
+
+compatible: "chipsemi,chsc6x"
+
+include: i2c-device.yaml
+
+properties:
+  irq-gpios:
+    type: phandle-array
+    required: true
+    description: |
+      The irq signal defaults to active low as produced by the
+      sensor. The property value should ensure the flags properly
+      describe the signal that is presented to the driver.
```

Comment:
This might have been copy-pasted from somewhere else but it looks like it's missing an actual description of the property?
See ex. https://docs.zephyrproject.org/latest/build/dts/api/bindings/input/focaltech%2Cft5336.html#dtbinding-focaltech-ft5336 or https://docs.zephyrproject.org/latest/build/dts/api/bindings/input/sitronix%2Ccf1133.html#dtbinding-sitronix-cf1133